tests/integration/api/test_aws_integration.py

```diff
@@ -0,0 +1,99 @@
+import os
+from datadog import api as dog
+from datadog import initialize
+
+API_KEY = os.environ.get("DD_TEST_CLIENT_API_KEY", "a" * 32)
+APP_KEY = os.environ.get("DD_TEST_CLIENT_APP_KEY", "a" * 40)
+API_HOST = os.environ.get("DATADOG_HOST")
+TEST_ACCOUNT_ID_1 = "123456789101"
+TEST_ACCOUNT_ID_2 = "123456789102"
+TEST_ACCOUNT_ID_3 = "123456789103"
+TEST_ACCOUNT_ID_4 = "123456789104"
+TEST_ROLE_NAME = "DatadogApiTestRole"
+TEST_ROLE_NAME_2 = "DatadogApiTestRolo"
+
+
+class TestAwsIntegration:
+
+    @classmethod
+    def setup_class(cls):
+        """ setup any state tied to the execution of the class. called at class
+        level before and after all test methods of the class are called.
+        """
+        initialize(api_key=API_KEY, app_key=APP_KEY, api_host=API_HOST)
+        dog.Aws.create(
+            account_id=TEST_ACCOUNT_ID_1,
+            role_name=TEST_ROLE_NAME
+        )
+        dog.Aws.create(
+            account_id=TEST_ACCOUNT_ID_2,
+            role_name=TEST_ROLE_NAME
+        )
```

Comment:
Why create these as part of a `setup_class` ? we could create it only in the test that requires. 
Right now, you're modifying those as part of the test, so the `teardown_class` doesn't delete the same things that you created, which is counter-intuitive. Plus if one of the tests modifying that resource don't pass, you won't be deleting the correct thing in the teardown.

Reply:
I did this to avoid having to create the same resource over and over in each method that would use it. I see what you mean though. I've added it to every method individually instead.